terraform/gitops/mojaloop/mcm.tf

```diff
@@ -75,6 +75,10 @@ module "generate_mcm_files" {
     mcm_service_account_name                 = var.mcm_service_account_name
     pki_client_role                          = var.pki_client_cert_role
     pki_server_role                          = var.pki_server_cert_role
+    pki_server_cert_ttl                      = var.pki_server_cert_ttl
+    pki_server_cert_max_ttl                  = var.pki_server_cert_max_ttl
+    pki_client_cert_ttl                      = var.pki_client_cert_ttl
+    pki_client_cert_max_ttl                  = var.pki_client_cert_max_ttl
```

Comment:
these only need to be passed in the `vault-pki-setup.tf` files, as the vault template is only referenced there:

<img width="707" height="731" alt="Image" src="https://github.com/user-attachments/assets/715450d1-df53-4837-a1d3-c17c41d36281" />